source/sections/footer.liquid

```diff
@@ -7,9 +7,11 @@
     {% form 'customer', class: 'form' %}
       {{ form.errors | default_errors }}
```

Comment:
i originally missed your ask about needing this for errors.

potentially.  It depends on where/how the errors are loaded in.  

optimally errors for inputs would be associated with said input by either including the error in the label associated with that label, or utilizing aria-describedby on the input, to point to the individual error message for said input.

a single 'error message' announcing that the form that was trying to be submitted has errors, could benefit from the aria-live approach as used for the success message above.  but each of the individual errors likely dont' need to be included via aria-live... at least from my current understanding of how the errors could/would work